### PR TITLE
Test image relocation in the pipeline

### PR DESCRIPTION
**Pull request by hoegaarden**
- Add separate taskcat tests to test relocation of CE & TAP images.
- Run those tests in concourse as part of the "main" and the "PR" pipeline
- opportunistically does some pipeline cleanup (removal of `old_name` for already renamed jobs)

### CODE REVIEW

**Comment by hoegaarden** [comment]
This adds the new test to the PR pipeline, which would prolong the PR pipeline for about 25min in the best case, much more in the worst case (in case we run into an error and need to wait for all wait conditions to time out).  
Do we maybe want to only run this test in the "main" pipeline, but not in the "PR" pipeline? What do you all think?

**Review by leclaist — APPROVED** [review]

File: .taskcat.yml [hunk 1]
```diff
@@ -30,11 +29,19 @@ tests:
...
     regions:
       - us-east-1
       - eu-south-1
   multi-cluster:
     parameters:
       TAPClusterArch: multi
+      TanzuNetRelocateImages: "No"
+    regions:
+      - us-west-2
+  relocation:
```
Comment (satya-dillikar) [inline .taskcat.yml]: Add multi-cluster-ecr and single-cluster-ecr. Basically two new tests.

**Review by satya-dillikar — CHANGES_REQUESTED** [review]

**Comment by satya-dillikar** [comment]
Add these new tests for both the main and pr pipelines.

**Comment by hoegaarden** [comment]
> Add these new tests for both the main and pr pipelines.

I wanted to not introduce too many tests, for "cost" reasons (actual $$$ cost, feedback time, and housekeeping time).
But happy to add those 2 test cases, esp. because its easy to be removed from our pipeline, but still have them run at release on AWS e2e check.

----

Thinking about this a bit more:
We could also decide to not test relocation on PR, but only run it on main, which runs daily anyway. Means, we wouldn't run "too much" on PR but would still get "async" feedback on the daily run on the against main. Just a thought ... we can also do this in a different PR, if we want to go that route.
(Ha, actually [thought about that already yesterday](https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/pull/92#issuecomment-1468868012) :smile:)

**Review by satya-dillikar — APPROVED** [review]

**Pull request merged**